### DISCUSSION TITLE
Multiple hard drives on server

### DISCUSSION THREAD

**Discussion by renatopinto89** [AzuraCast/AzuraCast#4194] [General]
Hey guys

I have a server with 2 drives, one system SSD and a attached 500GB HDD, I install and run AzuraCast just fine on the SSD and everything works as it should, but I can´t find a way to set the media to go to HDD instead, I´m probably missing a setting or something and I´m asking for your help on how to set that up.

Thank you

v0.12.4 Stable • Docker • PHP 8.0 - ubuntu-20.04

**Comment by ncfernan** [comment]
@renatopinto89  I couldn't locate the instructions on how to use another local storage location for media files, I guess they are rewriting them. The old instructions led to more than one interpretation. I took the wrong turn myself, but now it's working like a charm.

(seu nome sugere que você é brasileiro...)

**Comment by renatopinto89** [comment]
@ncfernan (Quase, português haha)

 I searched high and low and couldn't find any instructions and since my server only has 30GB of SSD storage that is a little short. I would really appreciate if you could share what you did to have that working. 

Thank you

**Comment by ncfernan** [comment]
@renatopinto89  mande um e-mail para ncfernan no gmail que eu te explico.

I can guide you, no problem.

**Reply by ucartz** [reply]
Got any solution ?

**Reply by ghost** [reply]
I have the same issue and haven't come up with a solution yet. I was told to start an issue! I would have thought that this isn't a new request and something other admins would have encountered and found a solution.

**Reply by ghost** [reply]
I found an easy fix for my server:
I logged in using ssh (you need root privileges)  changed to /var/azuracast/stations 
I created a symbolic link to the larger drive path looks like this:
christian -> /data/azuracast/stations/christian
in the AzuruaCast CP I created the station and uploaded files to the larger drive space.
Ubuntu 20.04LTS
Powered by • v0.16.1 Stable • Ansible • PHP 8.1

**Comment by BusterNeece** [comment]
If you want to store your station data on a hard drive separate from the one running your primary OS, follow these instructions to customize your Docker configuration and point your station media directory at the secondary drive: https://docs.azuracast.com/en/administration/docker#storing-your-station-data-on-the-host-machine
